webapp/src/App/theme.tsx

```diff
@@ -27,5 +27,8 @@ export default createMuiTheme({
     secondary: {
       main: '#ff9800',
     },
+    error: {
```

Comment:
the color already exist.
though, it's a good idea to create a new one for the "delete red", but not raw "red" (#ff0000) plz ^^